.circleci/config.yml

```diff
@@ -670,6 +669,32 @@ workflows:
                 service-name: "${AWS_RESOURCE_NAME_PREFIX_CODEDEPLOY_FARGATE}-service"
                 cluster-name: "${AWS_RESOURCE_NAME_PREFIX_CODEDEPLOY_FARGATE}-cluster"
 
+      - aws-ecs/deploy-service-update:
```

Comment:
If I recall correctly, we should remove the original `delete-service` invocation so that it doesn't cause `codedeploy_fargate_test-update-and-wait-service-job` to fail

Reply:
Thanks! I've removed that from the post_steps.